python/ray/experimental/state/common.py

```diff
@@ -193,8 +196,34 @@ class State(StateSchema):
     # Returns {"column_a", "column_b"}
     s.columns()
     ```
+
+    In addition, the schema also provides a humanify abstract method to
+    convert the state object into something human readable, ready for printing.
+
+    Subclasses should override this method, providing logic to convert its own fields
+    to something human readable, packaged and returned in a dict.
+
+    Each field that wants to be humanified should include a 'format_fn' key in its
+    metadata dictionary.
     """
 
+    @classmethod
+    def humanify(cls, state: dict) -> dict:
+        """Convert the given state object into something human readable."""
+        for f in fields(cls):
+            if f.metadata.get("format_fn") is not None and state[f.name] is not None:
+                state[f.name] = f.metadata["format_fn"](state[f.name])
+        return state
+
+    @classmethod
+    def is_valid_state(cls, state: dict) -> bool:
```

Comment:
we could remove this now?

Reply:
Oh I thought keeping it wouldn't hurt since it could be re-used as a useful util? What do you think?

Reply:
nah nvm, u know more than me, we don't need it! I can remove :P

Reply:
usually I think it' a good practice to avoid dead code in the codebase.

Reply:
Sounds good, thanks for letting me know. Will remember for future projects!